src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSpark.java

```diff
@@ -75,16 +67,45 @@
               optional  = true)
     public boolean enableSTDIOCapture = false;
 
-    // a few hard-coded parameters for use in various SGA modules based on some tuning experiences.
-    // subject to future changes as we see more test cases
-    @VisibleForTesting static final int MIN_OVERLAP_IN_FILTER_OVERLAP_ASSEMBLE = 55;
-    @VisibleForTesting static final int FILTER_STEP_KMER_FREQUENCY_THREASHOLD = 1;
-    @VisibleForTesting static final double OVERLAP_STEP_ERROR_RATE = 0.05;
-    @VisibleForTesting static final double ASSEMBLE_STEP_MAX_GAP_DIVERGENCE = 0.0;
-    @VisibleForTesting static final int ASSEMBLE_STEP_MIN_BRANCH_TAIL_LENGTH = 50;
-    @VisibleForTesting static final int CUT_OFF_TERMINAL_BRANCHES_IN_N_ROUNDS = 0;
-    @VisibleForTesting static final boolean TURN_ON_TRANSITIVE_REDUCTION = false;
-
+    //----------------------------------------------------------------------------------------------------------------//
+    // Result-affecting parameters for use in various SGA modules with default values.
+    // Long names are the same as those in SGA documentation, except using camel case in place of dashes.
+    //----------------------------------------------------------------------------------------------------------------//
+    @Argument(doc       = "Minimum overlap parameter value for steps: fm-merge, overlap, and assemble.",
```

Comment:
I think you also made these changes in the branch sh_sga_para_and_error_handle, no? I'd keep the changes in one branch and wait til it gets merged, otherwise you'll have trouble at rebase time.